pkg/components/rook-ceph/manifests.go

```diff
@@ -25,7 +25,7 @@ metadata:
   namespace: {{ .Namespace }}
 spec:
   cephVersion:
-    image: ceph/ceph:v15.2.4-20200630
```

Comment:
Generally ceph pushes image to same tag. So to make things predictable we should tag it to the specific version that rook docs suggests.

Reply:
Ok, I took this as an example:
https://github.com/rook/rook/blob/v1.4.6/cluster/examples/kubernetes/ceph/cluster.yaml

Reply:
Yes the same doc also talks about the specific tag.

Also the current version of PR worked I have tested it. But please make the tag specific.